packages/react-ui-core/src/Gmap/utils/mapEventHelpers.js

```diff
@@ -1,13 +1,19 @@
 export const handleEvent = (instance, name, props) =>
   event => props[name](event, props, instance)
 
-export const setupEvents = (instance, events, props) => {
+export const setupEvents = (instance, events, props, once) => {
   const eventNames = Object.keys(events)
-
+  const type = once ? 'addListenerOnce' : 'addListener'
+  const kvp = {}
```

Comment:
It looks like you had this function return an object of `MapsEventListener` s in order to have easy access to to them to be removed later, but, could we name this something better than `kvp`?

Reply:
Can you also add a test for this?